scalableminds/webknossos
# Reload histogram when reloading layer

I had to move around the code in the histogram saga a little bit, reviewing ignoring white space changes should help.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a dataset locally, remove some mag 1 wkw files on disk and reload the layer - the histogram should change

### Issues:
- fixes #4479 

------
(Please delete unneeded items, merge only when none are left open)
